src/app/modules/activities-management/components/create-activity/create-activity.component.ts

```diff
@@ -17,7 +16,7 @@ export class CreateActivityComponent {
   constructor(private formBuilder: FormBuilder) {
     this.activityForm = this.formBuilder.group({
       name: ['', Validators.required],
-      description: ['']
+      description: ['', Validators.required],
```

Comment:
I don't if the description is required in activity. @enriquezrene can help us.

Reply:
it is not.

Reply:
It is not, let's remove this validation @daros10

Reply:
done @enriquezrene